mecanum_drive_controller/src/mecanum_drive_controller.cpp

```diff
@@ -303,8 +304,11 @@ bool MecanumDriveController::on_set_chained_mode(bool /*chained_mode*/) { return
 controller_interface::CallbackReturn MecanumDriveController::on_activate(
   const rclcpp_lifecycle::State & /*previous_state*/)
 {
-  // Set default value in command
-  reset_controller_reference_msg(*(input_ref_.readFromRT()), get_node());
+  // Try to set default value in command.
+  // If this fails, then another command will be received soon anyways.
+  ControllerReferenceMsg emtpy_msg;
+  reset_controller_reference_msg(emtpy_msg, get_node());
+  input_ref_.try_set(emtpy_msg);
```

Comment:
```suggestion
  input_ref_.set(emtpy_msg);
```
Shouldn't it be a set in this case? As we are in the activate callback

Reply:
Yes, you are right. In this particular case, it is wiser to use a set, as you want the controller to start with a zero state. Moreover, it is hard to have it blocking, unless you keep spamming the topic even before the controller started

Reply:
but if the lock fails here, wouldn't it be very likely that the recently received command can be read with the try_get at the beginning of the update method?

Reply:
then this blocking set here has no use